weave-js/src/components/PagePanelComponents/Home/Browse3/feedback/FeedbackGrid.tsx

```diff
@@ -21,14 +23,22 @@ export const FeedbackGrid = ({
   weaveRef,
   objectType,
 }: FeedbackGridProps) => {
+  const {loading: loadingUserInfo, userInfo} = useViewerInfo();
+
   const {useFeedback} = useWFHooks();
   const query = useFeedback({
     entity,
     project,
     weaveRef,
   });
 
-  if (query.loading) {
+  const getTsClient = useGetTraceServerClientContext();
```

Comment:
any reason in particular you are directly using the tsClient here instead of putting the listener mount inside the useFeedback hook? Similar questions for `getTsClient().feedbackPurge({` in the `FeedbackGridActions.tsx` component? Maybe you prefer to skip the hooks interface all together?

Reply:
fwiw, i feel like refactoring / removing the wfHooks interface, so this comment is not an opposition

Reply:
If we do refactor I think it would be good if our API layer in general supported refetching and polling like we get with apollo and a standard error interface for every API.

Reply:
+1 - this totally agree